gov_uk_dashboards/components/plotly/header.py

```diff
@@ -19,13 +19,14 @@ def header(title: str, strong_class: str = "", background_colour: str = None):
             html.Div(
                 [
                     html.Img(
-                        src="assets\\images\\mhclg_white_no_background.png",
-                        srcSet="assets\\images\\mhclg_white_no_background.png 490w",
-                        sizes="(min-width: 600px) 200px, 30vw",
+                        src="assets\\images\\dcms_coatofarms.png",
+                        srcSet="assets\\images\\dcms_coatofarms.png 490w",
+                        sizes="(min-width: 600px) 400px, 30vw",
                         className="header-image",
-                        style={"maxWidth": "200px"},
-                        alt="Department for Levelling Up, Housing & Communities",
+                        style={"maxWidth": "400px"},
+                        alt="Ministry of Housing, Communities & Local Government",
                     ),
+                    html.Div(["Ministry of Housing, Communities & Local Government"]),
```

Comment:
Is the writing on the intranet a bit bolder? I was just comparing the PR screenshot and I think it might be?

Reply:
It looks like they might have a fixed size and font weight? 
![image](https://github.com/user-attachments/assets/5e125cf9-cd32-4b64-ace3-2afb5abe272c)